pyproject.toml

```diff
@@ -48,14 +48,16 @@ isort = "^5"
 pre-commit = "2"
 flake8 = "^4"
 mypy = "^0.931"
-cookiecutter = "^1.7.3"
 pytest-cov = "^3.0.0"
 pytest-cookies = "^0.6.1"
 Sphinx = "^4.4.0"
 Flake8-pyproject = "^0.9.0"
 jupyter-book = "^0.12.3"
 myst-parser = "^0.15"
 
+[tool.poetry.group.dev.dependencies]
+cookieninja = "1.0.0"
```

Comment:
thinking a bit about that .. cookieninja should not be a dev dep .. it should be a direct dependency

Reply:
Ok, I'm doing it now